modin/pandas/__init__.py

```diff
@@ -129,6 +130,11 @@ def _update_engine(publisher: Parameter):
     else:
         is_hdk = False
 
+    if Engine.get() == "Client":
```

Comment:
I think, this block could be moved to the very beginning of the method.

Reply:
Done.